Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/Test/TestAC.cs

```diff
@@ -47,6 +48,8 @@ public class TestAC
 
         public List<CategoryAC> CategoryAcList { get; set; }
 
+        public List<TestIPAddressAC> TestIPAddress { get; set; }
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename property 'TestIPAddress' to match camel case naming rules, consider using 'TestIpAddress'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS100)